### PR TITLE
Adding several community hand-wired dactyl manuform variants and wiring, as well as keymaps.

### PR DESCRIPTION
**Pull request by Lhun**
## Description
- adding personal and /r/mk community keymap examples,
    - Lhun keymap is for the fbr layout variant 5x6 by /u/brownelvis from r/mechanicalkeyboards
    - c# and にほんご friendly, reintroduces KC_LBRC KC_RBRC, LALT, LGUI, DEL and PLUS
    - inclusion of KC_MHEN,KC_HENK for microsoft ime jp 106+
    - _RAISE layer includes RALT for f-combo purposes
    - swapped sides for raise and lower to be windows combo aware

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/keymap.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* A standard layout for the Dactyl Manuform 5x6 Keyboard */ 
+
+#include QMK_KEYBOARD_H
+
+
+enum layers {
+    _BASE,
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/keymap.c:7]: This is preventing the keymap from compiling.

```suggestion
    _QWERTY,
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — CHANGES_REQUESTED** [review]
1. Please ensure all `.c` and `.h` files have license headers.
2. The layout macros added to `keyboards/handwired/dactyl_manuform/5x6/5x6.h` should have their layout data added to `keyboards/handwired/dactyl_manuform/5x6/info.json` as well.

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/rules.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/rules.mk:16]:
```suggestion
AUDIO_ENABLE = no           # Audio output
BLUETOOTH_ENABLE = no       # Enable Bluetooth
RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
```

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/rules.mk [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/rules.mk:3]:
```suggestion
#   change yes to no to disable
```

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-3thumbkeys/rules.mk [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-3thumbkeys/rules.mk:16]:
```suggestion
AUDIO_ENABLE = no           # Audio output
BLUETOOTH_ENABLE = no       # Enable Bluetooth
RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
```

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-3thumbkeys/rules.mk [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-3thumbkeys/rules.mk:3]:
```suggestion
#   change yes to no to disable
```

File: keyboards/handwired/dactyl_manuform/4x6/keymaps/brownelvis/keymap.c [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+),
+
+[_RAISE] = LAYOUT(
+    _______,RESET,  _______,_______,_______,KC_LBRC,            KC_RBRC,_______,KC_NLCK,KC_INS, KC_SLCK,KC_MUTE,   \
+    _______,KC_LEFT,KC_UP  ,KC_DOWN,KC_RGHT,KC_LPRN,            KC_RPRN,KC_MPRV,KC_MPLY,KC_MNXT,_______,KC_VOLU,   \
+    _______,_______,_______,_______,_______,_______,            _______,_______,_______,_______,_______,KC_VOLD,   \
+                                            _______,_______,    KC_EQL, _______,                                   \
+                                            _______,_______,    _______,_______,                                   \
+                                            _______,_______,    _______,_______,                                   \
+                                            _______,_______,    _______,_______                                    \
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/4x6/keymaps/brownelvis/keymap.c:78]:
```suggestion
[_BASE] = LAYOUT(
    KC_ESC,  KC_Q,  KC_W,   KC_E,   KC_R,   KC_T,               KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_MINS,
    KC_TAB,  KC_A,  KC_S,   KC_D,   KC_F,   KC_G,               KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,
    KC_LSFT, KC_Z,  KC_X,   KC_C,   KC_V,   KC_B,               KC_N,   KC_M,   KC_COMM,KC_DOT, KC_SLSH,KC_BSLASH,
                    KC_LBRC,KC_RBRC,                                            KC_PLUS,KC_EQL,
                                    RAISE,  KC_SPC,             KC_ENT, LOWER,
                                    KC_TAB, KC_HOME,            KC_END, KC_DEL,
                                    KC_BSPC,KC_GRV,             KC_LGUI,KC_LALT
),

[_LOWER] = LAYOUT(
    _______,_______,_______,_______,_______,KC_LBRC,            KC_RBRC, KC_P7, KC_P8,  KC_P9,  RESET,  KC_PLUS,
    _______,KC_HOME,KC_PGUP,KC_PGDN,KC_END ,KC_LPRN,            KC_RPRN, KC_P4, KC_P5,  KC_P6,  KC_MINS,KC_PIPE,
    _______,_______,_______,_______,_______,_______,            _______, KC_P1, KC_P2,  KC_P3,  KC_EQL, KC_UNDS,
                                            _______,KC_PSCR,    _______, KC_P0,
                                            _______,_______,    _______,_______,
                                            _______,_______,    _______,_______,
                                            _______,_______,    _______,_______
),

[_RAISE] = LAYOUT(
    _______,RESET,  _______,_______,_______,KC_LBRC,            KC_RBRC,_______,KC_NLCK,KC_INS, KC_SLCK,KC_MUTE,
    _______,KC_LEFT,KC_UP  ,KC_DOWN,KC_RGHT,KC_LPRN,            KC_RPRN,KC_MPRV,KC_MPLY,KC_MNXT,_______,KC_VOLU,
    _______,_______,_______,_______,_______,_______,            _______,_______,_______,_______,_______,KC_VOLD,
                                            _______,_______,    KC_EQL, _______,
                                            _______,_______,    _______,_______,
                                            _______,_______,    _______,_______,
                                            _______,_______,    _______,_______
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-fbr/keymap.c [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+#define SYS MO(_SYS)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_BASE] = LAYOUT_5x6_fbr(
+     KC_ESC , KC_1  , KC_2  , KC_3  , KC_4  , KC_5  ,                         KC_6  , KC_7  , KC_8  , KC_9  , KC_0  ,KC_MINS,
+     KC_TAB , KC_Q  , KC_W  , KC_E  , KC_R  , KC_T  ,                         KC_Y  , KC_U  , KC_I  , KC_O  , KC_P  ,KC_BSLASH,
+     KC_LSFT, KC_A  , KC_S  , KC_D  , KC_F  , KC_G  ,                         KC_H  , KC_J  , KC_K  , KC_L  ,KC_SCLN,KC_QUOT,
+     KC_LSFT, KC_Z  , KC_X  , KC_C  , KC_V  , KC_B  ,                         KC_N  , KC_M  ,KC_COMM,KC_DOT ,KC_UP,KC_SLSH,
+     OSM(MOD_LCTL),KC_LGUI,OSM(MOD_LALT),OSL(LOWER), KC_DEL, LOWER,           RAISE, KC_BSPC, OSL(RAISE), KC_LEFT, KC_DOWN, KC_RIGHT,
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis-fbr/keymap.c:24]:
```suggestion
     OSM(MOD_LCTL),KC_LGUI,OSM(MOD_LALT),OSL(LOWER), KC_DEL, LOWER,           RAISE, KC_BSPC, OSL(RAISE), KC_LEFT, KC_DOWN, KC_RIGHT,
```

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/keymap.c [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+  [_QWERTY] = LAYOUT_5x6_reversed(
+     KC_ESC , KC_1  , KC_2  , KC_3  , KC_4  , KC_5  ,                         KC_6  , KC_7  , KC_8  , KC_9  , KC_0  ,KC_MINS,
+     KC_TAB , KC_Q  , KC_W  , KC_E  , KC_R  , KC_T  ,                         KC_Y  , KC_U  , KC_I  , KC_O  , KC_P  ,KC_BSLASH,
+     KC_LSFT, KC_A  , KC_S  , KC_D  , KC_F  , KC_G  ,                         KC_H  , KC_J  , KC_K  , KC_L  ,KC_SCLN,KC_QUOT,
+     KC_LCTL, KC_Z  , KC_X  , KC_C  , KC_V  , KC_B  ,                         KC_N  , KC_M  ,KC_COMM,KC_DOT,KC_DOWN,KC_RIGHT,
+                      KC_PGUP,KC_PGDN,                                                       KC_LEFT, KC_UP,
+                                      RAISE,KC_SPC,                           KC_ENT, LOWER,
+                                      OSM(MOD_RCTL), KC_DEL,                  KC_BSPACE, OSM(MOD_RALT),
+                                      TG(LOWER), KC_LGUI,                     KC_HOME, KC_END
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/keymap.c:27]:
```suggestion
                                      TG(LOWER), KC_LGUI,                     KC_HOME, KC_END
```

File: keyboards/handwired/dactyl_manuform/5x6/keymaps/lhun-fbr/keymap.c [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+#define SYS MO(_SYS)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT_5x6_fbr(
+     KC_ESC , KC_1  , KC_2  , KC_3  , KC_4  , KC_5  ,                         KC_6  , KC_7  , KC_8  , KC_9  , KC_0  ,KC_MINS,
+     KC_TAB , KC_Q  , KC_W  , KC_E  , KC_R  , KC_T  ,                         KC_Y  , KC_U  , KC_I  , KC_O  , KC_P  ,KC_BSLASH,
+     KC_LSFT, KC_A  , KC_S  , KC_D  , KC_F  , KC_G  ,                         KC_H  , KC_J  , KC_K  , KC_L  ,KC_SCLN,KC_QUOT,
+     KC_LCTL, KC_Z  , KC_X  , KC_C  , KC_V  , KC_B  ,                         KC_N  , KC_M  ,KC_COMM,KC_DOT ,KC_UP,KC_SLSH,
+     KC_LGUI, KC_LALT,KC_LBRC,KC_RBRC, RAISE, KC_SPC,                        KC_BSPC, LOWER ,KC_PLUS, KC_LEFT, KC_DOWN, KC_RIGHT,
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keymaps/lhun-fbr/keymap.c:22]:
```suggestion
     KC_LGUI, KC_LALT,KC_LBRC,KC_RBRC, RAISE, KC_SPC,                        KC_BSPC, LOWER ,KC_PLUS, KC_LEFT, KC_DOWN, KC_RIGHT,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**